### PR TITLE
Fixing guest being removed from other not-visible user from a channel

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
When a guest was removed from a channel and the user wasn't visible, it needs
to ask for the user information to confirm that the other user is no longer
visible.

#### Ticket Link
[MM-19169](https://mattermost.atlassian.net/browse/MM-19169)

### CODE REVIEW

**Comment by jespino** [comment]
@migbot Sure

**Comment by jespino** [comment]
Done @migbot PTAL

File: actions/websocket_actions.jsx [hunk 1]
```diff
@@ -677,6 +678,10 @@ export function handleUserRemovedEvent(msg) {
             if (msg.data.remover_id === msg.broadcast.user_id) {
                 browserHistory.push(getCurrentRelativeTeamUrl(state));
             } else {
+                if (!getUser(state, msg.data.remover_id)) {
+                    await dispatch(loadUser(msg.data.remover_id));
+                }
+
                 const user = getUser(state, msg.data.remover_id) || {};
```
Comment (streamer45) [inline actions/websocket_actions.jsx]: Not a big deal but we could probably save a call to `getUser` if we restructure as such:
```js
let user = getUser(state, msg.data.remover_id);
                
if (!user) {
    await dispatch(loadUser(msg.data.remover_id));
    user = getUser(state, msg.data.remover_id) || {};
}
```

Comment (jespino) [reply actions/websocket_actions.jsx]: yes, good point

**Pull request merged**